src/global/right-click/right-click-system.js

```diff
@@ -104,12 +108,10 @@ function addRightClickOptions(systemTileObj) {
         const popupPanel = new VerticalBox();
         const popupUi = new UIElement();
         popupUi.widget = new Border().setChild(popupPanel);
-        popupUi.rotation = systemTileObj.worldRotationToLocal(
-            new Rotator(0, 0, 0)
-        );
+        popupUi.rotation = new Rotator(0, player.getRotation().yaw, 0);
```

Comment:
I think Wekker may also be making a change moving / rotating the popup.  No problem, just letting you know this might be replaced and/or conflict when that happens.